README.md

```diff
@@ -28,22 +70,58 @@ mtx.myTranspose();
        ]
 ```
 
-7. Use `myMap` to build a new array where every element is incremented by 10.
-8. Use `myMap` to build a new array where every non-string element is converted to an empty string. 
+7. Use `myMap` to build a new array where every element is incremented by
```

Comment:
Hey my friend! The more you attempt here, even if you don't feel confident, the more I can help, OK?